src/features/lightspeed/lightspeedUser.ts

```diff
@@ -135,7 +138,44 @@ export class LightspeedUser {
       }
     }
   }
+  /* Get the user info from server */
+  public async getMarkdownUserInfo(token: string) {
```

Comment:
Can you return a HTML string instead? This way you can rename the function `getUserdetailsSnippet()`.